### ISSUE TITLE
Should `ConstValue` add new constructor from bool ? 

### ISSUE THREAD

**Issue by mut0u** [abhinav/language-thrift#14]
I find that struct field with default bool value , it will parse the value to Text.

Example:

```
struct Demo{
  1: bool flag = false
}

```
It will generate `fieldDefaultValue` with the `ConstLiteral Text srcAnnot`. I think `ConstValue` should add `ConstBool Bool srcAnnot` constructor.

**Comment by abhinav** [comment]
You're correct, this was an oversight, but I'm no longer actively working on the project.
I'll be happy to review and merge a PR fixing this.
